CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -438,6 +444,24 @@ def main(argv):
         for key in wsMismatchDict.keys():
             print("%s : %s row%s mismatched" % (key, wsMismatchDict[key], "" if wsMismatchDict[key]==1 else "s"))
 
+
+def main(argv):
+    args = parseArguments()
+    try:
+        messages = validate_workbook(args.file, args)
+    except xl.exceptions.InvalidFileException as e:
+        print("Invalid File: %s" % (str(e),))
+        if args.debugMode:
+            tb.print_exc(e)
+        exit(-1)
+    except FatalError as e:
+        print(e.message)
+        exit(-1)
+    else:
```

Comment:
This else can be removed. The two exceptions exit from the program so you can remove this layer of indentation